docs/controls/AdaptiveGridView.md

```diff
@@ -10,23 +9,102 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, AdaptiveGridView,
 
 The **AdaptiveGridView Control** presents items in a evenly-spaced set of columns to fill the total available display space. It reacts to changes in the layout as well as the content so it can adapt to different form factors automatically.
 
-If there are not enough items to fill one row, the control will stretch the items until all available space is filled. This can result in much wider items than specified. If you prefer your items to always stay close to the DesiredWidth, you can set the **StretchContentForSingleRow** property to **false**, to prevent further stretching.
+There are 3 ways to use this Control. 
+
+* You can set `DesiredWidth` and `ItemHeight`, which will scale the **width** of each item, and adjust the number of columns on demand, using all horizontal space.
+
+* You can set `DesiredWidth` only. This will mean that rows will take up as much space as required, using all horizontal space.
+
+* Using `OneRowModeEnabled`, you can set `DesiredWidth` and `ItemHeight`, and the **width** of each item, and the number of visible columns will be adjusted on demand, using all horizontal space.
```

Comment:
sentence structure is strange here with the and.  I'd use the 'which will scale' that you used above.  And remove the comma after item again.